ando/tools/generator/AnDOGenerator.py

```diff
@@ -232,17 +239,23 @@ def generate_metadata_files(self):
                     create_file(mfile, parents[(3-level)] / mfile.name,
                                 mode='copy')
 
-    def validate(self):
+    def validate(self, output_folder):
```

Comment:
This is a method of `AnDOData`, so you can just use the self.basedir to run AnDO on.